blocks/components/editable/index.js

```diff
@@ -319,6 +388,27 @@ export default class Editable extends wp.element.Component {
 		}
 	}
 
+	isEmpty( value ) {
+		// Empty array.
+		if ( ! value.length ) {
+			return true;
+		}
+
+		const props = value[ 0 ].props;
```

Comment:
Yeah, I'm planning to revisit this. There seem to be a few cases before when there were paragraphs.